docs/book/deployment.md

```diff
@@ -389,6 +390,34 @@ that will be used to call Lambda's [updateFunctionConfiguration](
 You can use this to modify the Lambda's timeout, change the memory size, or
 add your own environment variables. All sub keys are optional.
 
+#### DynamoDB TTL (optional)
+
+Litexa has limited support for setting a [Time To Live (TTL)](
+  https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html)
+field with each database entry. This may be useful for limiting your table
+data to active users by automatically deleting old user data. To set a Time To
+Live specification, add the `dynamoDbConfiguration` object to your Litexa config:
+
+```json
+{
+  "deployments": {
+    "development": {
+      "module": "@litexa/deploy-aws",
+      "askProfile": "suncoast",
+      "awsProfile": "prototyping",
+      "dynamoDbConfiguration": {
+        "ttlName": "timeToLive", // the name of the TTL attribute to set
+        "ttlInSeconds": 2592000 // how many days (in seconds) the new/updated record lives for
+      }
+    }
+  }
+}
+```
+
+At this time, Litexa does not create or update your table with your TTL
+settings. You will need to manually activate the TTL attribute for your table
+in the AWS console or AWS CLI.
```

Comment:
There should be a way of automating this for full support through the AWS SDK (https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html)

If not, this snippet should be at the beginning with the link to the documentation on enabling it via the AWS console/CLI: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-how-to.html

Reply:
We can do it when we create the table, but it's actually a separate permission and API call. Litexa currently doesn't have any dynamo update scaffolding (in either core or deployment), so we wouldn't be able to support updates to the config like this. I'd like to hold off on doing that in this PR and group that into a larger dynamo->deploy-aws refactor.